CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,25 @@
+## General communication guidelines
+
+Ulauncher is a free, open source software project developed by a small team of volunteers and contributors.
+
+We are here because we want Ulauncher to improve, and for that we have to:
+* Act respectful towards each other.
+* Give each other the benefit of the doubt.
+* Give and accept constructive feedback.
+* Avoid arguments.
+* Learn from past mistakes.
+* Focus on what is best, not just for us as individuals, but for the Ulauncher community.
+* Refrain from trying to "win" arguments or advocate the importance of ones own specific needs.
```

Comment:
Advocating for your own specific needs is reasonable, doing so above all other needs or in spite of other needs, is not. So perhaps a potential rewording would be:

```
Refrain from elevating the importance of ones own specific needs above all other concerns.
```